tools/slurm-test.sh

```diff
@@ -54,13 +66,15 @@ scontrol show burstbuffer
 squeue
 
 echo "***Use persistent buffer***"
-adduser centos
+id centos &>/dev/null || adduser centos
 cat use-persistent.sh
 su centos -c 'sbatch --array=1-10 use-persistent.sh'
```

Comment:
So the problem is this test used to mount both persistent and per job, many times in parallel. I think we probably want to keep that, but we also want to keep your extra tests too. Maybe we need a "use-both.sh" or something with a better name :)

Reply:
I see yes that makes sense, I'll add one that uses both as well then. Thanks